conf.py

```diff
@@ -319,11 +319,11 @@ def xapian_code_example_command(ex, option):
     elif highlight_language == 'tcl':
         return "tclsh %s" % xapian_code_example_filename(ex)
     elif highlight_language == 'c++':
-        if option == 0:
-            return "g++ `xapian-config --cxxflags` %s code/c++/support.cc  -o %s `xapian-config --libs`\n./%s" \
+        if letor == False:
```

Comment:
It's generally regarded as better style not to compare boolean values against boolean literals, so `if not letor:` instead of `if letor == False:`.

(It's particularly problematic in languages without a real bool type, like C pre C99 - people would often make their own with something like `#define FALSE 0` and `#define TRUE 1` and `typedef int bool;` - problem is that a `bool` variable could end up with a value other than `0` or `1`, which is then true but not `TRUE` - `if (b)` is taken, but `if (b == TRUE)` isn't).